### PR TITLE
[ru] Localize `Edge computing`

### LINKED ISSUE #3057: [ru] Localize `Edge Computing`
### Description

This Issue is to localize [`Edge Computing`](https://github.com/cncf/glossary/blob/main/content/en/edge-computing.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
Create edge-computing.md and translate it into Russian

### Describe your changes

This PR is to add edge-computing.md and translate it into Russian.

### Related issue number or link (ex: `resolves #issue-number`)

resolves #3057 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/edge-computing.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+В некоторых случаях обработка данных в реальном времени не просто желательна, а жизненно необходима. 
+Достаточно вспомнить о самоуправляемых автомобилях. 
+Представьте, что данные с датчиков автомобиля приходилось бы передавать по сети в ЦОД для обработки, а затем обратно на автомобиль. 
+Подобная задержка в реакции, очевидно, неприемлема, когда дорога каждая секунда. 
+Хотя это доведенный до крайности пример, большинство пользователей не готовы ждать, когда их смартфон или иное умное устройство отреагирует на запрос, требуя мгновенной обратной связи.
+
+## Как именно решает проблему
+
+Как сказано выше, чтобы периферийные устройства были полезными, они должны выполнять хотя бы часть обработки и анализа локально, обеспечивая обратную связь почти в реальном времени. 
+Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД к месту, где эти данные генерируются: периферийному устройству.
```
Comment (shurup) [inline content/ru/edge-computing.md:25]:
```suggestion
Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД в место, где эти данные генерируются: периферийное устройство. 
```

File: content/ru/edge-computing.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Представьте, что данные с датчиков автомобиля приходилось бы передавать по сети в ЦОД для обработки, а затем обратно на автомобиль. 
+Подобная задержка в реакции, очевидно, неприемлема, когда дорога каждая секунда. 
+Хотя это доведенный до крайности пример, большинство пользователей не готовы ждать, когда их смартфон или иное умное устройство отреагирует на запрос, требуя мгновенной обратной связи.
+
+## Как именно решает проблему
+
+Как сказано выше, чтобы периферийные устройства были полезными, они должны выполнять хотя бы часть обработки и анализа локально, обеспечивая обратную связь почти в реальном времени. 
+Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД к месту, где эти данные генерируются: периферийному устройству. 
+Обработанные и необработанные данные затем отправляются в ЦОД для дальнейшей обработки и хранения. 
+Одним словом, эффективность и скорость — главная движущая сила пограничных вычислений.
```
Comment (shurup) [inline content/ru/edge-computing.md:27]: Perhaps my version is a bit less precise, but sounds better (IMHO) :sweat_smile: 
```suggestion
Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений. 
```

**Review by shurup — CHANGES_REQUESTED** [review]
We need to decide which translation we prefer to use as the primary one. Other corrections are optional.

File: content/ru/edge-computing.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Представьте, что данные с датчиков автомобиля приходилось бы передавать по сети в ЦОД для обработки, а затем обратно на автомобиль. 
+Подобная задержка в реакции, очевидно, неприемлема, когда дорога каждая секунда. 
+Хотя это и крайний пример, большинство пользователей не готовы работать с умными устройствами, которые не дают мгновенную обратную связь.
+
+## Как именно решает проблему
+
+Как сказано выше, чтобы периферийные устройства были полезными, они должны выполнять хотя бы часть обработки и анализа локально, обеспечивая обратную связь почти в реальном времени. 
+Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД в место, где эти данные генерируются: периферийное устройство. 
+Обработанные и необработанные данные затем отправляются в ЦОД для дальнейшей обработки и хранения. 
+Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений.
```
Comment (kirkonru) [inline content/ru/edge-computing.md:27]:
```suggestion
Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений.   
```

File: content/ru/edge-computing.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
```
Comment (kirkonru) [inline content/ru/edge-computing.md:27]:
```suggestion
Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений.
```

**Comment by shurup** [comment]
Now, I am confused why we have "Граничные вычисления" in the title but "Пограничные вычисления" at the very beginning of this article. Russian Wikipedia sticks to "Граничные" only. I suggest doing the same (consistently throughout the text).

File: content/ru/edge-computing.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Граничные (edge) вычисления
+status: Completed
+category: Technology
+---
+
+*Пограничные вычисления* (edge computing), также известные как *периферийные* — это подход к [распределённым системам](/ru/distributed-systems/), при котором часть задач по хранению и обработке данных переносится от основного [центра обработки данных](/ru/data-center/) к источнику данных.
```
Comment (kirkonru) [inline content/ru/edge-computing.md:7]:
```suggestion
*Граничные вычисления* (edge computing), также известные как *периферийные* — это подход к [распределённым системам](/ru/distributed-systems/), при котором часть задач по хранению и обработке данных переносится от основного [центра обработки данных](/ru/data-center/) к источнику данных. 
```

File: content/ru/edge-computing.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+Представьте, что данные с датчиков автомобиля приходилось бы передавать по сети в ЦОД для обработки, а затем обратно на автомобиль. 
+Подобная задержка в реакции, очевидно, неприемлема, когда важна каждая секунда. 
+Хотя это и крайний пример, большинство пользователей не готовы работать с умными устройствами, которые не дают мгновенную обратную связь.
+
+## Как именно решает проблему
+
+Как сказано выше, чтобы периферийные устройства были полезными, они должны выполнять хотя бы часть задач по обработке и анализу данных локально, обеспечивая обратную связь почти в реальном времени. 
+Это достигается за счёт переноса части ресурсов для хранения и обработки данных из ЦОД в место, где эти данные генерируются, то есть периферийное устройство. 
+Обработанные и необработанные данные затем отправляются в ЦОД для дальнейшей обработки и хранения. 
+Одним словом, эффективность и скорость — решающие факторы для пограничных вычислений.
```
Comment (kirkonru) [inline content/ru/edge-computing.md:27]:
```suggestion
Одним словом, эффективность и скорость — решающие факторы для граничных вычислений.
```

**Review by tym83 — APPROVED** [review]

**Review by shurup — APPROVED** [review]
Thank you; LGTM now!

**Pull request merged**